pipeline/fpm/fpm.go

```diff
@@ -99,6 +99,11 @@ func create(ctx *context.Context, format, folder, arch string, binaries []contex
 		options = append(options, "--conflicts", conflict)
 	}
 
+	// FPM requires --rpm-os=linux if your rpm target is linux
+	if format == "rpm" && strings.Contains(platform, "linux") {
```

Comment:
we can remove the `strings.Contains(platform, "linux")`, as right now this we only call this func when the platform is linux. Check:

https://github.com/goreleaser/goreleaser/blob/658d46397f6f6913d767987ab21c2de4d8ed9681/pipeline/fpm/fpm.go#L41

We can then also remove the additional `platform` parameter.